fix parallel compiling problem

without specifying lognormalizer_DEPENDENCIES = liblognorm.la,
lognormalizer won't link successfully with liblognorm.la as
liblognorm may not be generated yet.

Upstream-Status: Pending

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>
---
diff --git a/src/Makefile.am b/src/Makefile.am
index 8c15217..60f48c5 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -10,6 +10,7 @@ bin_PROGRAMS = lognormalizer
 lognormalizer_SOURCES = lognormalizer.c
 lognormalizer_CPPFLAGS =  -I$(top_srcdir) $(LIBEE_CFLAGS) $(LIBESTR_CFLAGS)
 lognormalizer_LDADD = $(LIBEE_LIBS) $(LIBLOGNORM_LIBS) $(LIBESTR_LIBS)
+lognormalizer_DEPENDENCIES = liblognorm.la
 
 lib_LTLIBRARIES = liblognorm.la
 
